### ISSUE TITLE
Current Page highlighted in Yellow Fails Colour contrast. 

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#13]
![image](https://cloud.githubusercontent.com/assets/15111528/10785824/9dbf87ec-7dba-11e5-8fa9-5db03b00aeef.png)

Only Colour that would fit from the TALA Colour palette is #231f20 (Black)
Otherwise, we should be able to go with some neutral colours like #000060.

Please check http://colorsafe.co/ for colour options.

**Comment by HerinHentry** [comment]
Can we select one of these Colours for the menu? 
Menu background #231f20
Active Colour #E86928
Inactive Colour #00AEB3

![image](https://cloud.githubusercontent.com/assets/15111528/10786709/caa9acba-7dbf-11e5-9019-72d28cba5ce9.png)

**Comment by mr-bobz** [comment]
Done
